controllers/apps/operations/switchover_util.go

```diff
@@ -274,7 +274,7 @@ func renderSwitchoverCmdJob(ctx context.Context,
 						Containers: []corev1.Container{
 							{
 								Name:            KBSwitchoverJobContainerName,
-								Image:           cmdExecutorConfig.Image,
+								Image:           cmdExecutorConfig.Exec.Image,
```

Comment:
should check cmdExecuteConfig.Exec not null ?

Reply:
It has been checked before.